### ISSUE TITLE
Support 1.4.4.9

### ISSUE THREAD

**Issue by martydingo** [Steffo99/lihzahrd#21]
Hiya there,

I have forked this code and incorporated another fork, and a PR here about optimizations & caching, and I have come up with a codebase that supports Terraria 1.4.4.9.

https://github.com/martydingo/lihzahrd/

Would you be interested in a PR, as so we can get the version in PyPi supporting Terraria 1.4.4.9?

**Comment by Steffo99** [comment]
Gladly! :D  

I didn't do it myself because I started a rewrite in Rust to get better performance (but then never finished it), and then when Journey's End dropped a friend said he'd contribute to it to get a timelapse of our server (but because of a misconfiguration world snapshots didn't save properly and so we forgot again)...

Open a PR when you're ready! :D

**Comment by martydingo** [comment]
I've set up the PR at https://github.com/Steffo99/lihzahrd/pull/22

> I didn't do it myself because I started a rewrite in Rust to get better performance 

Ah actually I noticed the performance, and at first I was a bit disappointed, to be honest. But actually, thinking this through, there's absolutely no reason why it needs to be quick. Any realtime processing is impossible due to how Terraria saves at a scheduled interval, as such the bottleneck lies in how often Terraria is configured to save, and scheduling this down to frequent saves per minutes seems like a super bad idea.   

I use this alongside the tShock REST API to draw player positions on top of a map from a world parsed by this on a scheduled interval, then drawn by a script invoking flyingsnake on a page refresh in Django. It's a little heavy but it's a PoC. 

TLDR performance isn't that much of an issue imho,
